src/main/java/io/ebean/enhance/common/EnhanceContext.java

```diff
@@ -95,6 +96,9 @@ public EnhanceContext(ClassBytesReader classBytesReader, String agentArgs, Agent
 
     this.transientInternalFields = getPropertyBoolean("transientInternalFields", manifest.isTransientInternalFields());
     this.checkNullManyFields = getPropertyBoolean("checkNullManyFields", manifest.isCheckNullManyFields());
+    if (getPropertyBoolean("printversion", true)) {
```

Comment:
I don't think we should bother with this but leave it up to application code ?

Reply:
How do you mean that? The application code (=code that gets enhanced at runtume) does not know, anything about the agent. 

So I thought, if run time enhancement is used and someone do not want that the agent prints the version, he/she can add `-javaagent:ebean-agent-${version}.jar=printVersion=false` as agent argument.

Reply:
Sorry, I wasn't clear there. I was thinking just default it to `false` ... so it only prints to sysout when explicitly requested.  

Thinking that framework code would use the static `Transformer.getVersion()` ... might be rare that we want the version in sysout.